lib/compress/zstd_compress.c

```diff
@@ -2177,6 +2192,125 @@ static int ZSTD_useTargetCBlockSize(const ZSTD_CCtx_params* cctxParams)
     return (cctxParams->targetCBlockSize != 0);
 }
 
+/* ZSTD_useBlockSplitting():
+ * Returns if block splitting param is being used
+ * If used, compression will do best effort to split a block in order to improve compression ratio.
+ * Returns 1 if true, 0 otherwise. */
+static int ZSTD_useBlockSplitting(const ZSTD_CCtx_params* cctxParams)
+{
+    DEBUGLOG(5, "ZSTD_useBlockSplitting(splitBlocks=%d)", cctxParams->splitBlocks);
+    return (cctxParams->splitBlocks != 0);
+}
+
+/* ZSTD_buildSequencesStatistics():
+ * Returns the size of the statistics for a given set of sequences, or a ZSTD error code,
+ * Also modifies LLtype, Offtype, MLtype, and lastNCount to the appropriate values.
+ */
+MEM_STATIC size_t
```

Comment:
minor:
`MEM_STATIC` is for `static` functions that _may_ not be invoked,
thus triggering an "unused function" warning.

I don't think it's the case here.
Actually, `ZSTD_buildSequencesStatistics()` _should_ be invoked,
otherwise it's a sign that something is going wrong.
Hence, prefer regular `static`.